echopype/calibrate/calibrate_ek.py

```diff
@@ -992,7 +992,7 @@ def compute_Sv(self, waveform_mode="BB", encode_mode="complex"):
         -------
         Sv : xr.DataSet
             A DataSet containing volume backscattering strength (``Sv``)
-            and the corresponding range (``range``) in units meter.
+            and the corresponding range (``echo_range``) in units meter.
```

Comment:
I think this is a nice example of using the "range" concept while also pointing to the name of the corresponding variable, `echo_range`